haskell/test/Lorentz/Contracts/Test/Permit.hs

```diff
@@ -739,3 +739,13 @@ permitSpec originate = do
                 (Call_FA2 $ FA2.Update_operators params)
               lCallEP stablecoinContract (Call @"Update_operators") params
               assertPermitCount stablecoinContract 1
+
+    describe "Set-expiry" $ do
+      it "can be accessed via a permit" $
+        integrationalTestExpectation $ do
+          withOriginated originate defaultOriginationParams $ \stablecoinContract -> do
+            withSender wallet1 $ do
+              pausePermitHash <- callPermit stablecoinContract testPauserPK testPauserSK 0 Pause
+              -- hash <- callPermit stablecoinContract testPauserPK testPauserSK 1 $ Set_expiry $ SetExpiryParam testPauser (0 , Just pausePermitHash)
```

Comment:
Is it supposed to be removed?

Reply:
How did you come across this? AFAIU, it is not in the final commit.

Reply:
Hm, right, seems to be removed.

> How did you come across this?

I was reviewing per-commit :shrug:

Reply:
>I was reviewing per-commit

Ah, makes sense. I ll keep it in mind. Do you want to rewrite commit history with more descriptive messages in this PR?